.github/secrel/config.yml

```diff
@@ -8,3 +8,7 @@ languages:
       # TODO PR #1549: remove this and test to see if it is still needed
       # Snyk does its own build to run SCA, so it needs the token to pull private artifacts
       - -PGITHUB_ACCESS_TOKEN=${{ secrets.ACCESS_TOKEN_GRADLE_BUILD }}
+  python:
+    version: 3.9
```

Comment:
According to https://github.com/department-of-veterans-affairs/abd-vro/blob/59ecb237d313cbab4d26ec362d61e8314417a1e9/domain-cc/README.md?plain=1#L9 `3.10` is being used.